bundles/org.openhab.binding.matter/src/main/resources/OH-INF/thing/channels.xml

```diff
@@ -0,0 +1,463 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<thing:thing-descriptions bindingId="matter"
+	xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+	xmlns:thing="https://openhab.org/schemas/thing-description/v1.0.0"
+	xsi:schemaLocation="https://openhab.org/schemas/thing-description/v1.0.0 https://openhab.org/schemas/thing-description-1.0.0.xsd">
+
+	<channel-type id="battery-voltage">
+		<item-type>Number:ElectricPotential</item-type>
+		<label>Battery Voltage</label>
+		<description>The current battery voltage</description>
+		<category>Energy</category>
+		<state pattern="%.1f %unit%" readOnly="true"/>
+	</channel-type>
+
+	<channel-type id="battery-alarm">
+		<item-type>String</item-type>
+		<label>Battery Alarm</label>
+		<description>The battery alarm state</description>
+		<category>Energy</category>
+		<state readOnly="true">
+			<options>
+				<option value="minThreshold">Below min threshold</option>
+				<option value="threshold1">Below threshold 1</option>
+				<option value="threshold2">Below threshold 2</option>
+				<option value="threshold3">Below threshold 3</option>
+				<option value="noThreshold">No battery alarm</option>
+			</options>
+		</state>
+	</channel-type>
+
+	<channel-type id="powersource-batpercentremaining">
+		<item-type>Number</item-type>
+		<label>Battery Percent Remaining</label>
+		<description>Indicates the estimated percentage of battery charge remaining until the battery will no longer be able
+			to provide power to the Node</description>
+		<category>Energy</category>
+		<tags>
+			<tag>Measurement</tag>
+			<tag>Energy</tag>
+		</tags>
+		<state readOnly="true" pattern="%.0f %%"/>
+	</channel-type>
+
+	<channel-type id="powersource-batchargelevel">
+		<item-type>Number</item-type>
+		<label>Battery Charge Level</label>
+		<description>Indicates a coarse ranking of the charge level of the battery, used to indicate when intervention is
+			required</description>
+		<category>Energy</category>
+		<tags>
+			<tag>Alarm</tag>
+			<tag>LowBattery</tag>
+		</tags>
+		<state readOnly="true">
+			<options>
+				<option value="0">Ok</option>
+				<option value="1">Warning</option>
+				<option value="2">Critical</option>
+			</options>
+		</state>
+	</channel-type>
+
+	<channel-type id="colorcontrol-color">
+		<item-type>Color</item-type>
+		<label>Color</label>
+		<description>The color channel allows to control the color of a light.
+			It is also possible to dim values and switch the
+			light on and off.
+		</description>
+		<category>ColorLight</category>
+		<tags>
+			<tag>Control</tag>
+			<tag>Color</tag>
```

Comment:
I am not sure we have `Color` property. To be checked as there were recent changes to semantic tags.
Same for `ColorTemperature` and `Brightness`.
If not, you can use `Light`.

Reply:
You have list here: https://github.com/openhab/openhab-core/blob/main/bundles/org.openhab.core.semantics/model/SemanticTags.csv

So, all your used tags do exist.